src/Strategy.js

```diff
@@ -0,0 +1,73 @@
+//@flow
```

Comment:
Should we call this module a `Strategy`? It seems that actual strategy is in `ElementPropSpec` module which defines which style props should be compiled as CSS and/or set as inline styles.